src/5.0_to_5.1/calculated-measures/migrateCalculatedMeasuresInDashboards.ts

```diff
@@ -0,0 +1,101 @@
+import {
+  ContentEntry,
+  ContentRecord,
+  CubeName,
+  DashboardState,
+  DataModel,
+  deserializeDashboardState,
+  MdxSelect,
+  serializeDashboardState,
+  WidgetWithQueryState,
+} from "@activeviam/activeui-sdk-5.0";
+import { produce } from "immer";
+import { migrateCalculatedMeasuresInMdx } from "./migrateCalculatedMeasuresInMdx";
+
+interface DashboardsFolder {
+  entry: ContentEntry;
+  children: {
+    thumbnails: ContentRecord;
+    content: ContentRecord;
+    structure: ContentRecord;
+  };
+}
+
+/**
+ * Takes a `ui/dashboards` folder from the content server, removes any calculated member definitions from the MDX and returns a `migratedDashboards` folder.
+ * Also returns the cubeName for each calculated measure found.
+ */
```

Comment:
We could just refer to the other function:
```suggestion
/**
 * Same as `migrateCalculatedMeasuresInWidgets` but for widgets within saved dashboards.
 */
```

It would also avoid saying "removes any calculated member definitions from the MDX" which is not misleading: we remove only those matching `namesOfCalculatedMeasuresToMigrate`